detox/src/invoke/EarlGrey.js

```diff
@@ -3,6 +3,18 @@ const instance = {
   value: 'instance'
 };
 
+function call(method, ...args) {
```

Comment:
I don't think we need to add a special `call` function here, we are calling EarlGrey's static methods elsewhere without the need for this code.

+ take a look at https://github.com/wix/detox/blob/master/detox/src/invoke/Invoke.js#L25
It's a neat trick using Proxy objects that enables this cool syntax:

```js
invoke.IOS.CGPoint({x: 2, y: 1}))
```
to actually work with no `CGPoint` function ever existing

https://github.com/wix/detox/blob/master/detox/src/invoke.test.js#L19